lib/numbers.md

```diff
@@ -5,20 +5,23 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Ruby has 'Fixnum' numbers, or 'intigers' that are whole numbers, and a 'float' wich a non whole number with a decimel point.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+I suppose the most common operation with numbers would be in a mathmatical equation to complete a code.  It will be necessary to find out what type of 
+number is being input by a user, and requesting that number be how the program needs it to be, like an integer or an even number
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+'+' when used with a string will help ruby to assign a value.  When its used with a number, it would simply add, or complete the mathmatical
```

Comment:
Just making sure you saw the above comment. I can approve this assignment once you address that question.